frontend/src/pages/CreateProfile/CreateProfileForm.js

```diff
@@ -59,82 +78,94 @@ function CreateProfileForm({ profile, setProfile, initialState, getUser }) {
       checkForProfile();
   }, [])
 
+  useEffect(() => {
+    setPfp(profile.image)
+    setUserAddress(profile.address)
+  
+    return () => {
+      console.log('profile image url: ', pfp)
+    }
+  }, [checkForProfile])
```

Comment:
I see what you're doing here.
A cleaner way to potentially do it is to make `profile` the dependency instead of the `checkForProfile` function.
Since inside the checkForProfile function you are calling setProfile, which updates profile, which would trigger this useEffect.

Reply:
I see what you mean, but would it be an issue if the form onChange (handleChange function) updates setProfile and triggers this useEffect every time profile state updates?

Reply:
Resolved this during call, decided that we'll refactor if we need to in the future.